### PR TITLE
Adding cva6 architecture on genesysII board

### PR DESCRIPTION
**Pull request by eyssartk**
This PR add support for the cva6 on genesysII board.

### CODE REVIEW

File: board/openhwgroup/cva6_genesysII/cva6.c [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#include <init.h>
+
+int board_init(void) {
+  debug("%s: could not enable cache ways\n", __func__);
```
Comment (niwis) [inline board/openhwgroup/cva6_genesysII/cva6.c]: Is there a condition for this statement, or will it always be printed?

**Review by niwis — COMMENTED** [review]
Thanks a lot for this contribution! Just one minor question regarding the `board_init` function.

File: board/openhwgroup/cva6_genesysII/cva6.c (hunk 1 above)
Comment (eyssartk) [reply board/openhwgroup/cva6_genesysII/cva6.c]: This is printed only if log or debug is activated. Indeed this message is useless here.

**Comment by eyssartk** [comment]
I just spotted an error in this PR. I encounter the same error as you Nils, it needs this commit in order to work:
https://github.com/eyssartk/u-boot/commit/b40b427db38bf0cea9ad1c3dd51fc478fac97071

**Review by niwis — APPROVED** [review]
Cool, nothing else to add from my side. Looks good to me!

**Comment by niwis** [comment]
I pushed the contributions to a new `cva6` branch to keep `master` aligned with upstream. (it might actually make sense to also open a PR there)

**Pull request merged**